internal/impl/devbox.go

```diff
@@ -611,6 +611,38 @@ func (d *Devbox) StartServices(ctx context.Context, serviceNames ...string) erro
 	return services.Start(ctx, d.packages(), serviceNames, d.projectDir, d.writer)
 }
 
+func (d *Devbox) StartProcessManager(ctx context.Context) error {
+	svcs, err := d.Services()
+	if err != nil {
+		return err
+	}
+	hasServiceWithProcessCompose := false
+	for _, s := range svcs {
+		if _, hasComposeYaml := s.ProcessComposeYaml(); hasComposeYaml {
+			hasServiceWithProcessCompose = true
+		}
+	}
+	if !hasServiceWithProcessCompose {
+		return usererr.New("No services to start")
+	}
+	if _, err := exec.LookPath("process-compose"); err != nil {
```

Comment:
I noticed that we search the PATH for it to be installed, but then we always use a Nix path when running it. Wouldn't this break if someone has process-compose installed outside of Nix?

Reply:
yeah, good call.